modules/vaos/app/services/vaos/v2/appointments_service.rb

```diff
@@ -570,6 +573,15 @@ def page_params(pagination_params)
       def date_format(date)
         date.strftime('%Y-%m-%dT%TZ')
       end
+
+      def validate_appointments_schema(response)
+        if response.success? && Flipper.enabled?(:schema_validation_appointments_index)
```

Comment:
I believe so, but we can cross that bridge when we get to it. At first I'll probably just turn it on in staging since the volume is much lower there and guarding on % probably won't be necessary.